test/cypress/support/index.ts

```diff
@@ -3,5 +3,9 @@
 import { configure } from '@testing-library/dom'
 
 import './commands'
+require('cypress-terminal-report/src/installLogsCollector')()
```

Comment:
Why are you using `require` instead of `import` as in the rest of the repo?

Reply:
Just copied from the readme to see if it worked 👀 Tomorrow I'll clean up this code. Marking it as a draft for now